### ISSUE TITLE
PDF parsing into text

### ISSUE THREAD

**Issue by RebeccaGreiner** [amosproj/amos2024ss05-knowledge-graph-extractor#4]
## User story
1. As a user,
2. I want uploaded PDF documents to be automatically parsed into text
3. so that the text is stored as a file.

## Acceptance criteria
* When a PDF is uploaded successfully, it should automatically be parsed into text.
* Error handling is implemented for parsing failures.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
